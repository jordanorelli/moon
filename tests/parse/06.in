a_key: this is a value
another_key: @a_key
@var_key: this shouldn't ever be exported outside of the moon interpreter
public_key: @var_key
