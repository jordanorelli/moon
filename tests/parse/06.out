root:
  assign:
    name:
      a_key
    value:
      string:
        this is a value
  assign:
    name:
      another_key
    value:
      variable:
        a_key
  assign:
    name:
      var_key
    value:
      string:
        this shouldn't ever be exported outside of the moon interpreter
  assign:
    name:
      public_key
    value:
      variable:
        var_key
